libfwupdplugin/fu-common-version.h

```diff
@@ -15,7 +15,8 @@ gchar		*fu_common_version_from_uint32	(guint32	 val,
 						 FwupdVersionFormat kind);
 gchar		*fu_common_version_from_uint16	(guint16	 val,
 						 FwupdVersionFormat kind);
-gchar		*fu_common_version_parse	(const gchar	*version);
+gchar		*fu_common_version_parse	(const gchar	*version)
+G_DEPRECATED_FOR(fu_common_version_parse_from_format);
```

Comment:
is it worth adding to `fu-deprecated.h` or anything?

Reply:
I don't think so -- it seems a bit harsh to move it to fu-common-version-private.h too, although nothing will be actually linking that symbol directly I hope.

Reply:
FYI: https://github.com/fwupd/fwupd/pull/1638